exercises/accumulate/accumulate.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Accumulate do
+  @doc """
+    Given a function and a list, apply the function to each list item and
+    replace it with the function's return value.
+
+    Returns a list.
+
+    ## Examples
+
+      iex> Accumulate.accumulate(fn(x) -> x * 2 end, [])
+      []
+
+      iex> Accumulate.accumulate(fn(x) -> x * 2 end, [1, 2, 3])
+      [2, 4, 6]
+
+  """
+
+  @spec accumulate(Enumerable.t, (any -> any)) :: list
+  def accumulate(fun, list) do
```

Comment:
`map` is implemented in ListOps. It also has a notice about not using any of the built-in Enum or List functions, which this exercise would also need, so that folks don't simply submit `def accumulate(list, fun), do: Enum.map(list, fun)`